.github/workflows/api-tests.yml

```diff
@@ -163,4 +163,16 @@ jobs:
         repo-token: ${{ secrets.ORG_GH_TOKEN }}
         allow-repeats: true
       env:
-        STATUS: "${{ steps.test_execution.outcome == 'success' && ':white_check_mark:' || ':no_entry_sign:' }}"
\ No newline at end of file
+        STATUS: "${{ steps.test_execution.outcome == 'success' && ':white_check_mark:' || ':no_entry_sign:' }}"
+       
+    - name: Xray update
+      if: ${{ always() }} && github.event_name != 'pull_request'
+      run: |
+        ./update_xray.sh
+      working-directory: tyk-automated-tests
+      env:
+        TEST: "QA-901"
```

Comment:
It appears to me that this will have to be looked up from a map of branchname -> jira ticket for the automatic sync to do the right thing else it will be the same ticket in all branches.

Reply:
the TEST variable is a link to tests instance - we are executing the same test on all branches.
Link to ticket that is grouping all executions per branch is defined as EXECUTION variable in the script https://github.com/TykTechnologies/tyk-automated-tests/blob/master/update_xray.sh

Reply:
Cool!